spec/models/model_test_train_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
1. Read that article more thoroughly -- this test should be testing the validations, not just testing that `create` works properly
2. rename this file to `train_spec.rb`